ChangeLog

```diff
@@ -11,6 +11,7 @@ Bugfix
      previously lead to a stack overflow on constrained targets.
    * Add `MBEDTLS_SELF_TEST` for the mbedtls_self_test functions
      in the header files, which missed the precompilation check. #971
+   * Fix memory leak in in mpi_miller_rabin()
```

Comment:
```suggestion
   * Fix memory leak in in mpi_miller_rabin(). Contributed by Jens Wiklander in #2363.
```
Unless you prefer the credit to be given to Linaro.

Reply:
Thanks for the suggestion. I added both. :-)